clients/go/tests/integration/createJob_test.go

```diff
@@ -0,0 +1,144 @@
+package integration
+
+import (
+    "encoding/json"
+    "fmt"
+    . "github.com/onsi/ginkgo"
+    . "github.com/onsi/gomega"
+    "github.com/zeebe-io/zeebe/clients/go"
+    "github.com/zeebe-io/zeebe/clients/go/commands"
+)
+
+type MyJob struct {
+    ID string
+    Name string
+}
+
+func (job MyJob) String() string {
+    return fmt.Sprintf("{\"name\": \"%s\"}", "myjob")
+}
+var _ = Describe("CreateJob", func() {
+
+    var client zbc.ZBClient
+    BeforeEach(func() {
+        c, e := zbc.NewZBClient("0.0.0.0:26500")
+        Expect(e).To(BeNil())
+        Expect(c).NotTo(BeNil())
+        client = c
+    })
+
+    AfterEach(func() {
+        client.Close()
+    })
+
+    Context("create job", func() {
+        It("should create one job", func() {
+            _, err := client.NewDeployWorkflowCommand().AddResourceFile("../../../java/src/test/resources/workflows/demo-process.bpmn").Send()
```

Comment:
Remove the deployment commands from the tests. To create a job no workflow is required. Let's keep the tests focused.